src/lti/contrib/django/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class NonceHistory(models.Model):
+    client_key = models.CharField(max_length=200)
+    timestamp = models.IntegerField()
+    nonce = models.CharField(max_length=200)
```

Comment:
I made this an Integer field because I wanted to keep look ups as fast as possible. I like the idea of using a cache instead, and agree that a clean up command is needed.